knexfile.js

```diff
@@ -0,0 +1,46 @@
+// Update with your config settings.
+
+module.exports = {
+  development: {
+    client: 'pg',
+    connection: {
+      host: 'localhost',
```

Comment:
Would recommend pulling these values from `process.env` to not commit sensitive data to your repo. Ex. `host: process.env.PG_HOST`